cmd/kubeadm/app/constants/constants.go

```diff
@@ -387,7 +387,7 @@ var (
 		10: "3.1.12",
 		11: "3.2.18",
 		12: "3.2.24",
-		13: "3.2.24",
+		13: "3.3.10",
```

Comment:
Please keep this as-is (v1.13.x should use 3.2), add a `3.3.10` record for v1.14.x, and remove the branches we don't care about anymore, `v1.10.x` and `v1.11.x`.

Reply:
Thanks. Fixed.